aiplatform/src/test/java/aiplatform/imagen/GenerateImageSampleTest.java

```diff
@@ -69,5 +71,14 @@ public void tearDown() {
   public void testGenerateImageSample() throws IOException {
     PredictResponse response = GenerateImageSample.generateImage(PROJECT, "us-central1", PROMPT);
     assertThat(response).isNotNull();
+
+    Boolean imageBytes = false;
+    for (Value prediction : response.getPredictionsList()) {
+      Map<String, Value> fieldsMap = prediction.getStructValue().getFieldsMap();
+      if (fieldsMap.containsKey("bytesBase64Encoded")) {
+        imageBytes = true;
+      }
+    }
+    assert (imageBytes);
```

Comment:
Same comment, this `assert` won't do anything, and won't assert anything. You should use JUnit's own assertion methods, or `Assertions`'s, or `Truth`'s methods.

Reply:
Updated to assertThat(imageBytes).isTrue()